libs/providers/openstack.py

```
@@ -131,13 +134,43 @@ def get_flavor_obj(self, vm_name: str) -> Any:
         )
 
     def get_volume_metadata(self, vm_name: str) -> Any:
```

Ruff (0.15.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `get_volume_metadata`

(ANN401)